terminus-ui/confirmation/src/confirmation.directive.ts

```diff
@@ -24,6 +27,26 @@ import { TsButtonComponent } from '@terminus/ui/button';
 import { TsConfirmationModalComponent } from './confirmation-modal.component';
 
 
+/**
+ * Define the accepted string values for the {@link TsConfirmationModalComponent} position
+ */
+export type TsConfirmationModalPositionTypes
+  = 'above'
+  | 'below'
+  | 'before'
+  | 'after'
```

Comment:
Not sure `left` and `right` are clearer than `before` and `after`?

Reply:
Yeah I think both would work. Material actually allows both types of phrasing. Here my vote is to stick with before/after simply because it matches our existing tooltip api.

Reply:
To echo Ben, I chose these options to match tooltip position. This nomenclature is better if we ever have to support RTL languages.